dg645Sup/dg645.db

```diff
@@ -0,0 +1,308 @@
+record(bo, "$(P)$(R)SIM")
+{
+    field(SCAN, "Passive")
+    field(DTYP, "Soft Channel")
+    field(ZNAM, "NO")
+    field(ONAM, "YES")
+    field(VAL, "$(RECSIM=0)")
+}
+
+record(bo, "$(P)$(R)DISABLE") 
+{
+    field(DESC, "Disable comms")
+    field(PINI, "YES")
+    field(VAL, "$(DISABLE=0)")
+    field(OMSL, "supervisory")
+    field(ZNAM, "COMMS ENABLED")
+    field(ONAM, "COMMS DISABLED")
+}
+
+## Trigger settings and readback + suplementary records for reading
+## T0 and T1 that are not implemented by Delaygen
+
+# Trigger set
+
+record(mbbo,"$(P)$(R)TriggerSource:SP")
+{
+    field(DESC, "Trigger source")
+    field(SCAN, "Passive")
+    field(ZRST, "Internal")
+    field(ONST, "External Rising Edges")
+    field(TWST, "External Falling Edges")
+    field(THST, "SS External Rising Edges")
+    field(FRST, "SS External Falling Edges")
+    field(FVST, "Single Shot")
+    field(SXST, "Line")
+    field(ZRVL, "0")
+    field(ONVL, "1")
+    field(TWVL, "2")
+    field(THVL, "3")
+    field(FRVL, "4")
+    field(FVVL, "5")
+    field(SXVL, "6")
+    field(DTYP, "Soft Channel")
+    field(OUT, "$(P)$(R)TriggerSourceMO")
+    field(FLNK, "$(P)$(R)TriggerSourceMO")
+    info(dgconfig, "VAL")
+}
+
+# Trigger actual
+
+record(mbbi,"$(P)$(R)T1ReferenceMI")
+{
+    field(PINI, "YES")
+    field(DESC, "Chan T1 Ref")
+    field(SCAN, "Passive")
+    field(ZRST, "T0")
+    field(ONST, "T1")
+    field(TWST, "A")
+    field(THST, "B")
+    field(FRST, "C")
+    field(FVST, "D")
+    field(SXST, "E")
+    field(SVST, "F")
+    field(EIST, "G")
+    field(NIST, "H")
+    field(ZRVL, "0")
+    field(ONVL, "1")
+    field(TWVL, "2")
+    field(THVL, "3")
+    field(FRVL, "4")
+    field(FVVL, "5")
+    field(SXVL, "6")
+    field(SVVL, "7")
+    field(EIVL, "8")
+    field(NIVL, "9")
+    field(DTYP, "asynInt32")
+    field(INP,  "@asyn($(PORT)) T1_REF")
+    field(SIML, "$(P)$(R)SIM")
+    field(SIOL, "$(P)$(R)SIM:T1ReferenceMI")
+    field(SDIS, "$(P)$(R)DISABLE")
+    field(ASG, "READONLY")
+    info(INTEREST, "HIGH")
+}
+
+record(ai,"$(P)$(R)T0DelayAI")
+{
+    field(PINI, "YES")
+    field(SCAN, "1 second")
+    field(DESC, "Chan T0 Delay")
+    field(PREC, "12")
+    field(EGU,  "Sec")

```

Comment:
Sec is not a valid unit, see https://github.com/ISISComputingGroup/EPICS-DbUnitChecker/blob/41299e8e7c08c671e7cd7239b1f82f8a49758d9e/utils/db_checks.py#L20 for valid units